### ISSUE TITLE
Intellisense eventually stops working, keyboard stops working - not sure if this extension or not

### ISSUE THREAD

**Issue by jeremy-holt** [vincaslt/vscode-highlight-matching-tag#26]
- Extension Name: highlight-matching-tag
- Extension Version: 0.4.4
- OS Version: Windows_NT x64 10.0.16299
- VSCode version: 1.19.2

 We have written the needed data into your clipboard. Please paste:
```json 
{
	"activationTimes": {
		"startup": true,
		"codeLoadingTime": 1,
		"activateCallTime": 0,
		"activateResolvedTime": 0,
		"activationEvent": "*"
	},
	"runtimeErrors": [
		{
			"name": "TypeError",
			"message": "Cannot read property 'closing' of null"
		}
	]
}
```

**Comment by mikevaleriano** [comment]
Had to uninstall.

The extension is incredibly useful, but it's making vscode unusable. =/

**Comment by vincaslt** [comment]
Should be fixed in v0.5.
